src/pages/messages/components/NviCandidatesList.tsx

```diff
@@ -48,10 +48,10 @@ export const NviCandidatesList = ({
             <>
               <List data-testid={dataTestId.tasksPage.nvi.candidatesList} disablePadding sx={{ mb: '0.5rem' }}>
                 {nviCandidatesQuery.data?.hits.map((nviCandidate, index) => {
-                  const offsetNextCandidate = (page - 1) * rowsPerPage + index + 1;
+                  const candidateIndex = (page - 1) * rowsPerPage + (index + 1);
```

Comment:
lurer på om offset er et bedre navn enn index på denne, for det er vel offset i søket det er snakk om her

Reply:
jepp, det har du helt rett i!